onebusaway-android/src/main/java/org/onebusaway/android/io/elements/ObaRegionElement.java

```diff
@@ -72,6 +72,45 @@ public double getLonSpan() {
         }
     }
 
+    public static class Open311Servers implements ObaRegion.Open311Servers {
```

Comment:
If this object represents a single Open311 server, it should be `Open311Server`.